app/livechat/server/methods/saveInfo.js

```diff
@@ -22,11 +22,10 @@ Meteor.methods({
 		check(roomData, Match.ObjectIncluding({
 			_id: String,
 			topic: Match.Optional(String),
-			tags: Match.Optional(String),
+			tags: Match.Optional([String]),
```

Comment:
Would not require a migration here?

Reply:
It's not required because we're now passing this parameter as an array and handling this change inside `Livechat.saveRoomInfo`.